data/json/items/generic/filament.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "id": "thread",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "thread",
+    "description": "A small quantity of thread that could be used when sewing.",
+    "weight": 1,
+    "volume": "1ml",
```

Comment:
When I set `volume` to `1000 ml`, `stackable` to `true` and `stack_size` to `200` it gives thread enormous volume in-game (like 8 liters for 8 sinews), so volume should be defined per single item.